docs/user-guide/kubectl/kubectl_annotate.md

```diff
@@ -84,6 +84,7 @@ kubectl annotate pods foo description-
 ```
       --all[=false]: select all resources in the namespace of the specified resource types
   -f, --filename=[]: Filename, directory, or URL to a file identifying the resource to update the annotation
+      --include-extended-apis[=true]: If true, include definitions of new APIs via calls to the API server. [default true]
```

Comment:
Do we need a flag for this? I'd prefer not to add this, esp. if we're defaulting true.